diffsims/crystallography/reciprocal_lattice_point.py

```diff
@@ -360,6 +356,20 @@ def calculate_theta(self, voltage):
         wavelength = get_refraction_corrected_wavelength(self.phase, voltage)
         self._theta = np.arcsin(0.5 * wavelength * self.gspacing)
 
+    def _raise_if_no_point_group(self):
```

Comment:
How often do you anticipate using these methods? I would be tempted to either write the code explicitly where it's used, or rename them `check_point_group_present` or similar? Might need more thinking time on it.